docs/en_us/internal/testing.rst

```diff
@@ -610,6 +616,24 @@ During acceptance test execution, Django log files are written to
 
 **Note**: The acceptance tests can *not* currently run in parallel.
 
+
+Testing internationalization with dummy translations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any text you add to the platform should be internationalized. To generate
+translations for your new strings, run the following:
```

Comment:
Add a noun: "... run the following command."
(Our style is also to use complete sentences when introducing lists and code examples, so a period at the end of the sentence rather than leaving it open with a colon.)